### PR TITLE
update sites in small multiples to match map

### PR DESCRIPTION
**Pull request by auremoser**
hey @acabunoc,
I added some small multiples to match the map additions that arliss and I made today. Let me know if it's o.k. to merge.

* UK
* South Africa
* Netherlands

**Screenshot of the local version running**
new-maps

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -0,0 +1,568 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (acabunoc) [inline public/js/main.js]: Hey @auremoser, I think you commit this file by mistake! This is generated on each run so it shouldn't be in the repo

Comment (acabunoc) [inline public/js/main.js]: Hey @auremoser, I think you commit this file by mistake! This is generated on each run so it shouldn't be in the repo

**Comment by acabunoc** [comment]
Other than the extra file added, this looks good to me!

File: public/js/main.js (hunk 1 above)
Comment (auremoser) [inline public/js/main.js]: Should it be in the .gitignore then? I only commited the `views/events/globalsprint2016.jade` file so I'm not sure how this snuck in. I'll remove and recommit but let me know if there's a better workflow for this?

**Comment by auremoser** [comment]
O.k. removed it!

**Comment by acabunoc** [comment]
Sweet! R+ 👍 

merge away!

File: public/js/main.js (hunk 1 above)
Comment (acabunoc) [inline public/js/main.js]: Ooohh, yes it should be in .gitignore! Good idea.

**Pull request merged**